PolicyAndKeys-Lib/TestRequests.cs

```diff
@@ -82,14 +82,20 @@ public bool CheckAndGenerateTest(ref string id, ref string content)
             return false;
         }
 
+        private static Random random = new Random();
+        private static string RandomString(int length)
+        {
+            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+            return new string(Enumerable.Repeat(chars, length)
+              .Select(s => s[random.Next(s.Length)]).ToArray());
+        }
         private void CheckBeforeCreateTestKeyset()
         {
             if (TestKeysetID == null)
             {
                 var json = Constants.CreateKeyset;
-                var guid = Guid.NewGuid().ToString();
 
-                json = json.Replace(Constants.KEYSETID_TOKEN, guid);
+                json = json.Replace(Constants.KEYSETID_TOKEN, RandomString(8));
```

Comment:
WE could have just got an input name from the customer

Reply:
i know i am trying to make it easier for somebody to just test it.